src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -10,38 +10,52 @@ public class Et1VettingService {
     static final String ET1_DOC_TYPE = "ET1";
     static final String ACAS_DOC_TYPE = "ACAS Certificate";
     static final String DOC_LINK_DEFAULT = "/cases/case-details/%s#Documents";
-    static final String BEFORE_LINK_LABEL = "Open these documents to help you complete this form: "
-            + "<br/><a target=\"_blank\" href=\"%s\">ET1 form (opens in new tab)</a>"
-            + "<br/><a target=\"_blank\" href=\"%s\">Acas certificate (opens in new tab)</a>"
+    static final String BEFORE_LABEL_TEMPLATE = "Open these documents to help you complete this form: %s%s"
             + "<br/>Check the Documents tab for additional ET1 documents the claimant may have uploaded.";
+    static final String BEFORE_LABEL_ET1 =
+            "<br/><a target=\"_blank\" href=\"%s\">ET1 form (opens in new tab)</a>";
+    static final String BEFORE_LABEL_ACAS =
+            "<br/><a target=\"_blank\" href=\"%s\">Acas certificate %s (opens in new tab)</a>";
+    static final String BEFORE_LABEL_ACAS_OPEN_TAB = "<br/><a target=\"_blank\" href=\"%s\">"
+            + "Open the Documents tab to view/open Acas certificates (opens in new tab)</a>";
 
     /**
      * Update et1VettingBeforeYouStart.
      * @param caseDetails Get caseId and Update caseData
      */
-    public void initialBeforeYouStart(CaseDetails caseDetails) {
+    public void initialiseEt1Vetting(CaseDetails caseDetails) {
+        caseDetails.getCaseData().setEt1VettingBeforeYouStart(initialBeforeYouStart(caseDetails));
```

Comment:
I don't think this method is required, why not just setEt1VettingBeforeYouStart directly from the controller method?